### ISSUE TITLE
[Bug] Loading spinner is invisible on an inverted button

### ISSUE THREAD

**Issue by NickeManarin** [jgthms/bulma#2914]
### Overview of the problem

This is about the Bulma **CSS framework**, version 0.7.5 and newer.

Browsers:
- Microsoft Edge 44.18362.449.0  
- Firefox 75.0
- Chrome 80.0.3987.163

### Description

When using a button with `is-primary is-inverted is-loading`, the loading icon spinner is invisible.

Similar issues: #1686 and #470

### Steps to Reproduce

Go to the https://bulma.io/documentation/elements/button/#styles page and set one of the inverted buttons to "is-loading".

### Expected behavior

The spinner should be visible.

![Loading](https://user-images.githubusercontent.com/14798947/79486280-d9203f00-7fec-11ea-9693-f40f11044428.gif)

### Actual behavior

The spinner is not visible.

![image](https://user-images.githubusercontent.com/14798947/79486429-11c01880-7fed-11ea-950a-410476c94640.png)

**Comment by cblauvelt** [comment]
It appears that the fix in commit f19ac155342d5bd5a64c8d4174c460ee98065070 that was applied for `is-outlined` would also apply here.

In the meantime I added this to the app's scss to make it work, more or less:
```
.button.is-inverted.is-loading {
    &:hover,
    &:focus,
    &:active,
    &:visited {
      &::after {
        border-color: transparent transparent $grey $grey !important;
      }
    }
  }
```

**Comment by NickeManarin** [comment]
Bump...
